src/docs/GetStarted/Designers/whatsIncluded/index.mdx

```diff
@@ -7,24 +7,26 @@ The UI Kit is a set of sketch files to be used as libraries.
 
 It contains all **core styles** as well as a collection of **reusable components** based on those core styles.
 
->We recommend that you read the [Naming convention](/GetStarted/Designers/namingConvention/) page to understand the naming convention used on all sketch files.
+> We recommend that you read the [Naming convention](/GetStarted/Designers/namingConvention/) page to understand the naming convention used on all sketch files.
 
-***
+---
 
 ## Styles :
-* [AA] Global Styles.sketch
-  * Color Styles
-  * Text Styles
-  * Link Styles
-  * Shadows
 
-* [AA] Icons.sketch
+- [AA] Global Styles.sketch
+
+  - Color Styles
+  - Text Styles
+  - Link Styles
+  - Shadows
 
+- [AA] Icons.sketch
 
 ## Components :
-* [BA] Buttons.sketch
-* [BA] Forms.sketch
-  * Inputs
+
+- [BA] Buttons.sketch
+- [BA] Forms.sketch
+  - Inputs
 
```

Comment:
It's not up to date, these is the current list :

## Styles :

- [AA] Aspect rations.sketch
- [AA] Fonts.sketch
- [AA] Icons.sketch
- [AA] Shadows.sketch
- [AA] Colors.sketch

## Components :

- [BA] Bodies.sketch
- [BA] Buttons.sketch
- [BA] Forms.sketch
- [BA] Headings.sketch
- [BA] Heros.sketch
- [BA] Links.sketch
- [BA] Ratings.sketch
- [BA] Tags.sketch
- [BA] Flags.sketch
- [BB] Notifications.sketch